### ISSUE TITLE
Question about decay constant

### ISSUE THREAD

**Issue by yangchenyun** [slaypni/SM-15#3]
Hi,
   I am wondering how do you calculate the decay constant for a power approximation at [L278](https://github.com/slaypni/SM-15/blob/master/sm.coffee#L278)? 
   I checked on [wiki](http://en.wikipedia.org/wiki/Exponential_decay) that only exponential function will have a decay constant, how do you approach to compute one for a fixed point power function?
   Thanks again :+1: 

Best,

**Comment by slaypni** [comment]
Hmm... it seems mysterious.
I am sorry that I do not remember well about its implementation.

What I remember is that I made some inferences for implementing __ofm_ because [SuperMemo website](http://www.supermemo.com/help/smalg.htm) did not provide enough information for it. So it is not necessary to think that this code is the correct way to do it.

Perhaps, it was just a bug (−_−). It seems to be that _b_ at [L288](https://github.com/slaypni/SM-15/blob/master/sm.coffee#L288) should be approximately the same to the one of [L277](https://github.com/slaypni/SM-15/blob/master/sm.coffee#L277), however, the implementation does not seem to be. I could not remember my intention well.

**Comment by yangchenyun** [comment]
Oh man, I could totally imagine what happened when you were working on it.. The website lacks some "important" pieces to people who want to implement it.. 

Btw, are you still using supermemo to these days?

**Comment by slaypni** [comment]
Yeah, it was even not easy to grasp the overview of the algorithm when reading the website first time.

I have created [a web service](http://slaypni.github.io/flashward/) for memorizing foreign words (particularly English for me). I am still using it occasionally where SM.js is used inside of it.

**Comment by yangchenyun** [comment]
Yes, I've read some related paper by the same author to grasp the main idea and uses your implementation as guidelines.

I used to have more than 3000+ self-created cards on supermemo but after I migrated my OS to *nix, it becomes harder to use them and I am very unhappy for the fact that they keep user's data inside the software and there is not way I could easily inspect my own "model of brain".

Do you aware of this: http://supermemopedia.com/wiki/Algorithm_SM-17

**Comment by slaypni** [comment]
Oh, I did not read related papers. It could have helped me to implement the code if I knew it.

Perhaps, it may be good to if there are some way to log our activity related to memorization. Then we can compare multiple brain models to choose the one which suit to us. It could be more nice if there was a repository of the logs collected from many persons for developing a new algorithm.

Is that a new SuperMemo algorithm!? Wow, I was not aware of it. Although I did not read it, it seems to be getting better...

**Comment by yangchenyun** [comment]
I thought so, the accuracy and converge rate of the algorithm relies heavily on the the initial data set we assumes for the user. 

Yeah, that's the new algorithm when I send an email to the supermemo team and ask them questions about implementing sm-15 algorithm; they send me this link about their new endeavor.

**Comment by linonetwo** [comment]
Hi, a comment of sm fan (one of my friend) is here at zhihu https://www.zhihu.com/question/500098895/answer/2283656621
